### PR TITLE
add const keyword node and addresses #766

### PR DESCRIPTION
**Pull request by VictorHom**
relates to: https://github.com/romefrontend/rome/issues/766

- adds support for a TS const node

### CODE REVIEW

File: packages/@romefrontend/ast/js/unions.ts [hunk 1]
```diff
@@ -8,7 +8,10 @@
 import * as n from "../index";
 import {OptionalProps} from "@romefrontend/typescript-helpers";
 
-export type AnyTSEntityName = n.JSReferenceIdentifier | n.TSQualifiedName;
+export type AnyTSEntityName =
+	| n.JSReferenceIdentifier
+	| n.TSQualifiedName
+	| n.TSConstKeyword;
```
Comment (sebmck) [inline packages/@romefrontend/ast/js/unions.ts:14]: I don't think this node should be in this union and is one of the reasons for the CI failure, was something breaking without it?

Comment (VictorHom) [reply packages/@romefrontend/ast/js/unions.ts:14]: I have to double check

**Comment by VictorHom** [comment]
if you look at the snapshots, I don't have the node wrapped in the `TSTypeReference`. Should the `TSConstKeyword` be inside `TSTypeReference`?

**Comment by sebmck** [comment]
Awesome! Looks great. Sorry about taking so long to review.

**Pull request merged**